pkg/sql/catalog/resolver/resolver.go

```diff
@@ -301,7 +300,7 @@ func ResolveSchemaNameByID(
 	ctx context.Context,
 	txn *kv.Txn,
 	codec keys.SQLCodec,
-	dbID descpb.ID,
+	db catalog.DatabaseDescriptor,
```

Comment:
Plumbing the db descriptor came from a desire of removing a catalogkv function but had the nice side-benefit of reducing round trips to KV.